dataplicity/app.py

```diff
@@ -3,12 +3,13 @@
 
 import argparse
 import logging
+import logging.config
 import sys
 
 from . import __version__
 from . import subcommand
+from .subcommands import *
```

Comment:
Is there a need for the wildcard import?

Reply:
No, you're right. Better to be explicit.